code/game/objects/items/rogueweapons/melee/swords.dm

```diff
@@ -92,6 +99,10 @@
 	reach = 2
 	peel_divisor = 5
 
+/datum/intent/sword/peel/weak
```

Comment:
Add the unique peel attack verb to this. Specifically, we've already got a 'weakly peels' and 'strongly peels', on top of standard peeling. So perhaps better wording for this would do.